data/remnant/remnant ships.txt

```diff
@@ -892,37 +892,43 @@ ship "Merganser"
 		category "Interceptor"
 		licenses
 			Remnant
-		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"cost" 4830000
+		"shields" 3500
+		"hull" 3000
```

Comment:
1. The cool breeze isn't obtainable
2. The Merganser nerfs bring it above the 50 ton threshold of having hull > shields, so shields > hull is fine

Reply:
In that case, shields > hull makes more sense than hull > shields

Reply:
It's not a T1.5, the whole point is to NOT do the same as every other faction, integrated stuff is NOT 1:1 to equivalent regular outfits, and the cool breeze is an imaginary ship with no bearing or connection to anything.

I'd originally wanted to use damage resistance/protection instead of extra hp, but people said that's exactly the same thing as giving it hp, so don't use resistances/protection and just give it the equivalent hp.

Reply:
I am comparing it to other remnant ships. My cutoff isn't based on humans or wanderers or anything but the remnant. 

> After looking at shield/hull values and comparing to mass, it looks like 50 tons is the cutoff between hull > shield and shield > hull for remnant ships, so shield > hull is acceptable now as the merganser weighs 90 tons.
Merganser (current) 3000/5000 (30 mass)
Tern 500/2500 (32 mass)
Puffin 800/1500 (40 mass)
Petrel 1000/2500 (50 mass)
Smew 1200/2800 (50 mass)
Penguin 3000/1500 (71 mass)
Merganser (new) 3500/3000 (90 mass)